driver/normalizer/normalizer.go

```diff
@@ -31,8 +31,9 @@ var Preprocessors = []Mapping{
 
 func mapIASTNameDerived(typ string) Mapping {
 	return MapSemantic(typ, uast.Identifier{}, MapObj(
-		Obj{
-			"Name": Var("name"),
+		Fields{
+			{Name: "Name", Op: Var("name")},
+			{Name: "ExpandedFromMacro", Optional: "optMacro", Op: Any()},
```

Comment:
It's not used because it's dropped :) It's there any other way to tag a field as optional without using the `Optional` field?

Reply:
Not really. I can add `Drop: true` to `Fields`.

Reply:
https://github.com/bblfsh/sdk/pull/369

Reply:
That was fast!

Reply:
I'm confused - is something expected to be changed with this annotation regarding to latest SDK release with `Drop: true`?

Reply:
At least it could be, yes :)

Reply:
Also after reading https://github.com/bblfsh/sdk/pull/369/files#diff-3bd9e60cee55dd92c09d4e11f0b3905bR754 still confused by intent of
> tag a field as optional without using the `Optional` field

and the difference between `Optional + Any()` vs `Drop` (presumably `+Any()`)

Reply:
The only difference is that it won't declare a state variable as `Optional` does:
> Optional is implied, but the variable won't be created in this case.

Reply:
so `Optional` = `if there is X, you HAVE to use X in ... otherwise still match but do nothing` where the `Drop` is more like a promise that we would never use or need it in a target tree?